core/pfe-core/controllers/floating-dom-controller.ts

```diff
@@ -12,22 +11,26 @@ import {
   offset as offsetMiddleware,
   shift as shiftMiddleware,
   flip as flipMiddleware,
+  arrow as arrowMiddleware
 } from '@floating-ui/dom';
 
 type Lazy<T> = T|(() => T|null|undefined);
 
 interface FloatingDOMControllerOptions {
   content: Lazy<HTMLElement>;
   invoker?: Lazy<HTMLElement>;
-  arrow?: boolean;
+  arrow?: Lazy<HTMLElement>;
```

Comment:
@nikkimk 
Would you please take a look at the api changes here? Do you think they're exposing anything we could cover over? Would they be straightforward to migrate to s different underlying library?

Reply:
It should be fine.